md/csrf-security.md

```diff
@@ -73,3 +73,17 @@ To activate the addition of the secure flag, edit the configuration file and cha
 [User authentication overview](user-authentication-overview.md)
 [Read more about CSRF attacks](http://www.acunetix.com/websitesecurity/csrf-attacks)
 
+
+## How to use UI form or page from 7.3.3 or lower to 7.4.0 or greater when CSRF security is enabled
+To use a UI page or form who that was created in lower version that 7.4, you need to re-import this old page in actual UI designer. If you don't update your page, any API call in your outdated version will be failed because `X-Bonita-API-Token` was will be absent in response header. 
```

Comment:
remove the "who" in "To use a UI page or form who that was "
Change "lower version that 7.4" to "lower version than 7.4"
Change "was will be absent in response header." for "  will be absent in the response header."